backend/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.8.12-slim-buster
+
+WORKDIR /usr/src/app
+
+# Install necessary build dependencies
+RUN apt-get update && apt-get install -y build-essential
+
+COPY backend/requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
```

Comment:
do we need frontend? I would do a `COPY backend .` ?

Reply:
Yes, we need to skip frontend
But we also need to take other folders from the outside like misc folder
I'll change it to skip frontend